docs/easing.md

```diff
@@ -359,7 +359,7 @@ A useful tool to visualize cubic bezier curves can be found at http://cubic-bezi
 ### `in()`
 
 ```jsx
-static in(easing);
+static in easing;
```

Comment:
This is a Prettier issue, can we [ignore](https://prettier.io/docs/en/ignore.html#markdown) this line?

Reply:
This seems like a pretty big deal. How many other places is it doing something/going to do something similar?

Reply:
I was bit surprised but it seems that only this line is affected. 

I assume that the reason is that `in` in JS is a `keyword` and you usually cannot use it as a name for stand-alone variables or functions. Here it's fine because it's `static` method.